src/data/markdown/docs/30 xk6-browser/01 xk6-browser/02 BrowserContext/close.md

```diff
@@ -0,0 +1,6 @@
+---
+title: 'close()'
+excerpt: 'Close the BrowserContext and all its pages.'
+---
+
+Close the `BrowserContext` and all its [page](/javascript-api/xk6-browser/page)s. The `BrowserContext` is unusable after this call and a new one must be created. This is typically called to cleanup before ending the test.
```

Comment:
Do we need to add a code example here too?

Reply:
Resolved in: 938e08855aaf6e2b256cd4a7e192ea9a6aa5f5d5